CMakeLists.txt

```diff
@@ -117,6 +115,37 @@ target_include_directories(musica
 
 add_subdirectory(src)
 
+#
+# jiwon 7/21 - in progress
+#
+####################
+# MUSICA - fortran
+add_library(musica-fortran STATIC src/fortran_interface/micm/interface.F90)
+add_library(musica::musica-fortran ALIAS musica-fortran)
+
+set_target_properties(musica-fortran PROPERTIES
+  ARCHIVE_OUTPUT_DIRECTORY ${MUSICA_LIB_DIR}
+  Fortran_MODULE_DIRECTORY ${MUSICA_MOD_DIR}
```

Comment:
It's possible we'll want these to be new cmake variables, but we will figure that out when we make an example project

Reply:
Comments Added for this